src/utils.js

```diff
@@ -4,23 +4,29 @@ const jsonata = require('jsonata');
  * Checks member for a 'required' tag
  *
  * @param {Object} member A property definition
- * @returns {Boolean}
  */
-const hasRequiredTag = (member) => {
+const hasRequiredTag = async (member) => {
 	// Find any tag field whose `title` is 'required' (won't be there if not required)
-	const expression = "$[title='required']";
-	const result = jsonata(expression).evaluate(member.tags);
+	let result;
+	await (async () => {
+		const expression = "$[title='required']";
+		result = await jsonata(expression).evaluate(member.tags);
+	})();
```

Comment:
i think we can discard the anonymous async  function since the parent function is already async . please do the same in all 3 cases

Reply:
working example:

const hasRequiredTag = async (member) => {
	// Find any tag field whose `title` is 'required' (won't be there if not required)
	const expression = "$[title='required']";
	const result = await jsonata(expression).evaluate(member.tags);

	return !!result;
};